### PR TITLE
11.0 mig partner contact gender

### PR DESCRIPTION
**Pull request by jprobineau**

### CODE REVIEW

**Comment by yajo** [comment]
Setting as WIP until partner_contact_personal_information_page is merged

**Comment by jprobineau** [comment]
Module partner_contact_personal_information_page is in issue #511

**Comment by simahawk** [comment]
on my side: same remarks reported on #511 ;)

**Comment by simahawk** [comment]
partner_contact_personal_information_page merged w/ https://github.com/OCA/partner-contact/pull/507
@jprobineau can you update?

**Comment by njeudy** [comment]
@jprobineau did you work on this ? I can help or do migration

File: partner_contact_gender/tests/test_partner_contact_gender.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# © 2016 Therp BV <http://therp.nl>
+# © 2017-Apertoso N.V. (<http://www.apertoso.be>)
+#  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp.tests.common import TransactionCase
```
Comment (nikul-serpentcs) [inline partner_contact_gender/tests/test_partner_contact_gender.py:4]: Replace `openerp` to `odoo`.

**Comment by NL66278** [comment]
I think this can be closed in favour of https://github.com/OCA/partner-contact/pull/662#discussion_r236211284. Because this PR seems to be abandoned.

**Pull request closed without merging**